src/dashboard/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "trulens-dashboard" %}
+{% set version = "1.2.7" %}
```

Comment:
Can you update the other `meta.yaml`s too if you're planning on using this for the conda channel.